### DISCUSSION TITLE
Using Phidgets REL1100_0

### DISCUSSION THREAD

**Discussion by CryptoBroke550** [artisan-roaster-scope/artisan#1375] [Q&A]
Hello,  I setup the PWM command in Artisan using a slider, it is working but in 10% steps, can I change the step size? Want to use 5% steps if possible? or change if required?
Using:
Artisan 2.8.4
HUB0001_0
REL1100_0
PWM command out(0,{},s/n:0)

Thanks.

**Comment by MAKOMO** [comment]
You did not attach your artisan-settings.aset thus we do not know your configuration. Could it be that you set your slider steps to 10? The REL1100 supports 0-100% in steps of 1.

**Reply by CryptoBroke550** [reply]
Hello Makomo, when I do a windows search I don't see artisan-settings.aset, is this the Date.alog file? what file do I look for? Now I think I found the step button, it's on "1" and moves 10 on the slider, I get the same for 5 and 10. I'm new to Artisan, sorry.

It was in the windows documents folder.
Opps was not saving the settings correctly so I lost my changes but still need to know what I did wrong to only get 10 steps when Step set to 1? If I have that right?

**Reply by MAKOMO** [reply]
You can save your settings in a file `artisan-settings.aset` using menu Help >> Save Settings. Rename the file to end in `.txt` and drag-drop it here on a comment field on the GitHub Web page.

**Reply by CryptoBroke550** [reply]
I have re-created the slider, moves 10 steps. is this the line with the issue? step100temp=@Invalid()
The full file.
[artisan-settings-air.txt](https://github.com/artisan-roaster-scope/artisan/files/13867440/artisan-settings-air.txt)

**Reply by MAKOMO** [reply]
You are using an outdated version of Artisan (v2.8.4). Current version is 2.10.

I tested your settings on 2.8.4 and 2.10 and that air slider moves for me from 0 to 100 in steps of 1 as configured. Could you record a short screen video demonstrating the slider issue (should be independent of the Phidget PWM Command, thus you can also remove that setting).

Btw, our output command `out(0,{},s/n:0)` is wrong.  Not sure how this can work for you. Instead of the `s/n` there should be a proper serial number. Maybe you just removed it before posting here? Anyhow, you should be good with just `out(0,{})`, assuming you attach your fan to port 0 of the REL1100.

For 5% steps, you keep min/max to 0 and 100 and choose 5 under Steps.
